### PR TITLE
Collection roles with a few improvements

### PR DESCRIPTION
**Pull request by piotr-orwat-wttech**

### CODE REVIEW

File: roles/base/README.md
Comment (krystian-panek-wttech) [inline roles/base/README.md:1]: this is needed by ansible collection release process / external validator

File: roles/commons/.travis.yml
Comment (krystian-panek-wttech) [inline roles/commons/.travis.yml:1]: why travis? I don't like this

File: roles/instance/tests/clean.yml
Comment (krystian-panek-wttech) [inline roles/instance/tests/clean.yml:1]: why have you deleted my roles that I am actively using to perform quick tests on my Mac?

File: roles/author/.travis.yml
Comment (krystian-panek-wttech) [inline roles/author/.travis.yml:1]: Travis is passe ;)

File: examples/controller/src/ansible/requirements.yml [hunk 1]
```diff
@@ -10,5 +10,8 @@ collections:
     version: 1.2.0
   - name: community.general
     version: 2.4.0
-  - name: wttech.aem
-    version: 1.0.35
+  # - name: wttech.aem
+  #   version: 1.0.35
+  - name: https://github.com/wttech/aemc-ansible.git
+    type: git
+    version: improved-roles
```
[the remark this answers is not in the document]
Comment (piotr-orwat-wttech) [reply examples/controller/src/ansible/requirements.yml:17]: just for testing without a need of releasing the collection to galaxy-ansible

File: roles/author/.travis.yml
Comment (piotr-orwat-wttech) [reply roles/author/.travis.yml:1]: sorry, it comes from initial role template, will be removed

File: roles/instance/tests/clean.yml
Comment (piotr-orwat-wttech) [reply roles/instance/tests/clean.yml:1]: it has been actually splitted into two separate roles for managing author and publish separately

**Comment by krystian-panek-wttech** [comment]
@piotr-orwat-wttech as agreed, closing this PR, please reorganize the work:

1) prepare a separate PR in this repo with strictly AEMC module changes (*.py specifically)
2) consider introducing a separate repo with more production-ready AEM roles as aemc-ansible repo is intended to have example usage of modules; not to introduce best practices (due to ease of maintenance of the project); maintaining the AEM best practices is a pretty huge topic :)

**Pull request closed without merging**